etc/profile-m-z/tor-browser.profile

```diff
@@ -11,5 +11,7 @@ noblacklist ${HOME}/.tor-browser
 mkdir ${HOME}/.tor-browser
 whitelist ${HOME}/.tor-browser
 
+whitelist ${HOME}/.local/opt/tor-browser
```

Comment:
This needs `mkdir`.

Also `blacklist` it in `disable-common.inc`.

Reply:
Thanks @rusty-snake . I also noticed in other places the proflie that uses `mkdir` to create the directory, but I didn't understand why. Also, could you explain why need to `blacklist` it in `disable-common.inc`?

Reply:
> Also, could you explain why need to blacklist it in disable-common.inc?

To prevent other sandboxes (with blacklisting profiles) from accessing it.

Reply:
@rusty-snake Updated according to your suggestion, could you check if it was inserted to correct place?

Reply:
Just tested it but adding:

```
# tor-browser
blacklist ${HOME}/.local/opt/tor-browser
```

in etc/inc/disable-common.inc didn't work. Any idea?

Reply:
You must `noblacklist` it in tor-browser.profile.

Reply:
I've reverted blacklist in `disable-common.inc` for now until further suggestion.

Reply:
Thank you @rusty-snake it worked. I've updated the commit.

Reply:
Move it to to the other `noblacklist`, then I can :heavy_check_mark: it.

Reply:
> Move it to to the other noblacklist,

@rusty-snake Sorry I didn't understand what you meant, could you explain it more?

Reply:
@rusty-snake Not sure exact what you meant by:

> Move it to to the other noblacklist

But I guessed it and made the change, could you check it?